Test suite is broken on Angular 2.0.0-rc.4

```
src/polymer-element.spec.ts(5,1): error TS2305: Module '"angular2-polymer/node_modules/@angular/core/testing"' has no exported member 'injectAsync'.
src/polymer-element.spec.ts(14,10): error TS2305: Module '"angular2-polymer/node_modules/@angular/platform-browser/testing"' has no exported member 'dispatchEvent'.
```

--- a/src/polymer-element.spec.ts
+++ b/src/polymer-element.spec.ts
@@ -2,16 +2,13 @@ import {
 describe,
 expect,
 it,
-injectAsync,
 inject,
-async,
 setBaseTestProviders
 } from '@angular/core/testing';
 import { PolymerElement } from './polymer-element';
 import { TestComponentBuilder, ComponentFixture} from '@angular/compiler/testing';
 import { Component } from '@angular/core';
 import { ControlGroup, Control } from '@angular/common';
-import { dispatchEvent } from '@angular/platform-browser/testing';
 import { By } from '@angular/platform-browser/src/dom/debug/by';
 
 import {

